lib/provisioning/ProvisionDispatcher.js

```diff
@@ -306,14 +327,14 @@ class ProvisionDispatcher {
         );
     }
 
-    _onOwnerChange() {
+    _onOwnerChange(autoTrigger) {
         const zkPath = OWNERS;
         this._withRandomDelayIfTest(
-            () => this._client.getChildren(zkPath, event => {
+            () => this._myGetChildren(autoTrigger, zkPath, event => {
                 this._log.debug('got owner event',
                                 { zkPath: `${this._zkEndpoint}${zkPath}`,
                                   event });
-                this._onOwnerChange();
+                this._onOwnerChange(autoTrigger);
```

Comment:
if I understand correctly, here `autoTrigger` can only be true so for clarity we could pass `true` instead (same for other similar cases)